cellmlmanip/model.py

```diff
@@ -599,18 +599,19 @@ def _set_variable_type(variable, variable_type):
             logger.warning('Variable %s already has type=="%s". Skip setting "%s"',
                            variable.dummy, variable.type, variable_type)
 
-    def get_symbols(self, expr):
-        """Returns the symbols in an expression. Works differently to Sympy atoms() because a
+    def get_symbols(self, expressions):
+        """Returns the symbols in a collection of expressions. Works differently to Sympy atoms() because a
         Derivative instance is regarded as a single symbol"""
         symbols = set()
```

Comment:
Second issue with this method: A sympy derivative is not a symbol

https://github.com/sympy/sympy/blob/master/sympy/core/function.py#L1024

It's an expression!

Reply:
Updated the comment to reflect this